src/DssSpell.sol

```diff
@@ -55,100 +63,81 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        // https://forum.makerdao.com/t/mip55c3-sp3-ethamsterdam-event-spf/13781/74?u=patrick_j
-        DssExecLib.sendPaymentFromSurplusBuffer(ETH_AMSTERDAM_WALLET, 50_000);
+        // ----------------- Core Unit Budgets -----------------
+        // https://mips.makerdao.com/mips/details/MIP40c3SP37#budget-implementation
+        DssExecLib.sendPaymentFromSurplusBuffer(GRO_001_WALLET, 474_683);
+
+        // -------------- Fund Ambassador Program --------------
+        // https://vote.makerdao.com/polling/QmPpQ49p#poll-detail
+        DssExecLib.sendPaymentFromSurplusBuffer(AMBASSADOR_WALLET, 25_000);
+
+        // ---- Replace Gelato Keeper Top Up Contract Stream ---
+        // https://forum.makerdao.com/t/update-to-the-gelato-keeper-network-top-up-contract/14524
+        // Yank existing stream
+        MCD_VEST_DAI.yank(36);
+        // Replace stream
+        // Address: 0x478c7Ce3e1df09130f8D65a23AD80e05b352af62
+        // Amount: 1,000 DAI/day
+        // Start Date: Apr 1, 2022
+        // End Date: Sep 30, 2022 23:59:59 UTC
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                GELATO_WALLET_NEW,
+                183_000 * WAD,
+                APR_01_2022,
+                SIX_MONTHS,
+                0,
+                address(0)
+            )
+        );
+
+        // ------------------ Offboard TUSD-A -------------------
+        // https://vote.makerdao.com/polling/QmVkRdjg#poll-detail
+        bytes32 _ilk  = bytes32("TUSD-A");
+        address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_TUSD_A");
+        //
+        // Enable liquidations for TUSD-A
+        // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
+        //       the rely for consistency with other collaterals and in case the PIP
+        //       changes to an OSM.
+        DssExecLib.authorize(_clip, DssExecLib.clipperMom());
+        DssExecLib.setValue(_clip, "stopped", 0);
+        // Use Abacus/LinearDecrease
+        DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_TUSD_A);
+        // Set Liquidation Penalty to 0
+        DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
+        // Set Liquidation Ratio to 150%
+        DssExecLib.setIlkLiquidationRatio(_ilk, 15000);
+        // Set Auction Price Multiplier (buf) to 1
+        DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 10000);
+        // Set Local Liquidation Limit (ilk.hole) to 5 million DAI
+        DssExecLib.setIlkMaxLiquidationAmount(_ilk, 5 * MILLION);
+        // Set tau for Abacus/LinearDecrease to 21,600,000 second (estimated 10bps drop per 6 hours = 250 days till 0)
+        DssExecLib.setLinearDecrease(MCD_CLIP_CALC_TUSD_A, 21_600_000);
+        // Set Max Auction Duration (tail) to 432,000 seconds (5 days, implies minimum price of 0.98)
+        DssExecLib.setAuctionTimeBeforeReset(_ilk, 432_000);
+        DssExecLib.setAuctionPermittedDrop(_ilk, 9800);
```

Comment:
Note that this is not strictly needed as it is derived for the auction length. It was also not explicitly mentioned in the post but I see no reason why not to add it for extra safety as you did (so just writing this in case soemone does).

Reply:
I'll add the following comment.
https://discord.com/channels/893112320329396265/897483518316265553/963575806653767720

Reply:
Ah ok haven't seen that discussion.
Note that the channel is not accessible to everyone, but not sure that is super-critical.